src/prpy/rave.py

```diff
@@ -235,3 +235,229 @@ def disable_padding(body, enable=False):
     for link in body.GetLinks():
         if link.GetName().startswith('padding_'):
             link.Enable(enable)
+
+
+def add_box(env, name, x_len=0.1, y_len=0.1, z_len=0.1):
+    """Add simple box kinbody to OpenRAVE environment"""
+    with env:
+       box = openravepy.RaveCreateKinBody(env, '')
+       success = box.InitFromBoxes(numpy.array([[0,0,0,x_len,y_len,z_len]]))
+       box.SetName(name)
+       env.Add(box)
```

Comment:
It would be helpful to expose the (optional) `anonymous` argument here. This tells OpenRAVE to generate a unique name for the `KinBody` if it would otherwise be a duplicate.